src/lib/rest/StringFilter.cpp

```diff
@@ -633,13 +609,72 @@ bool StringFilterItem::parse(char* qItem, std::string* errorStringP, StringFilte
 
 /* ****************************************************************************
 *
+* lhsFix - change dots for equals, then remove all quotes
+*/
+static char* lhsFix(char* s)
+{
+  char* scopyP        = strdup(s);
+  char* dotP          = scopyP;
+  bool  insideQuotes  = false;
+  
+  //
+  // Replace ',' for '=' if inside quotes
+  //
+  while (*dotP != 0)
```

Comment:
We already have a function for . -> = transformation `dbDotEncode()`. Let's use it.

(FYI, we also have another function for the inverse transformation, `dbDotDecode()`)